subprojects/internal-testing/src/main/groovy/org/gradle/util/TestPrecondition.groovy

```diff
@@ -130,6 +130,9 @@ enum TestPrecondition implements org.gradle.internal.Factory<Boolean> {
     JDK12_OR_LATER({
         JavaVersion.current() >= JavaVersion.VERSION_12
     }),
+    JDK13_OR_EARLIER({
```

Comment:
If we can avoid it, I think we should try to not add this

Reply:
Managed to avoid this one finally!